public/util/tagManagerApi.js

```diff
@@ -57,9 +85,10 @@ export default {
   },
 
   createTag: (tag) => {
+    const clean = cleanTag(tag);
```

Comment:
Should this be done on update rather than create and save? So the client side is using the object that will be sent to the server?